CHANGELOG.md

```diff
@@ -6,6 +6,42 @@ section below.
 
 ### Unreleased changes
 
+- **⚠️ DEPRECATION**: Using the `prefix: "foo"` argument for `StatsD.metric`
+  calls (and the metaprogramming macros) is deprecated.
+
+  - You can simply include the prefix in the metric name.
+  - If you want to override the global prefix, set `no_prefix: true` and
+    include the desired prefix in the metric name
+
+  This library ships with a Rubocop rule to detect uses of this keyword
+  argument in your codebase:
+
+  ``` sh
+  # Check for the prefix arguments on your StatsD.metric calls
+  rubocop --only StatsD/MetricPrefixArgument \
+    -r `bundle show statsd-instrument`/lib/statsd/instrument/rubocop.rb
+  ```
+
+  Strict mode has also been updated to no longer allow this argument.
+
+- **⚠️ DEPRECATION**: Using the `as_dist: true` argument for `StatsD.measure`
+  and `statsd_measure` methods is deprecated. This argument was only available
+  for internal used, but was exposed in the public API. It is unlikely that you
+  have this enabled, but you can check it using this Rubocop invocation:
```

Comment:
By "enabled" are you referring to the rubocop rule?  Or do you actually mean to say it is unlikely this keyword argument was used?